php/libraries/Candidate.class.inc

```diff
@@ -625,9 +625,9 @@ class Candidate
         $query  = "SELECT Visit_label
             FROM session s JOIN candidate c ON (c.CandID = s.CandID)
             WHERE c.CandID=:cid AND Date_Visit IS NOT NULL
-            ORDER BY Date_Visit";
+            ORDER BY Date_Visit LIMIT 1";
```

Comment:
do not need 'LIMIT 1' LIMIT clause is implemented in pselectRow (called from pselectOne)